ts/screens/authentication/cie/CieCardReaderScreen.tsx

```diff
@@ -177,6 +178,8 @@ class CieCardReaderScreen extends React.PureComponent<Props, State> {
   private choosenTool = assistanceToolRemoteConfig(
     this.props.assistanceToolConfig
   );
+  private navigation = this.props.navigator.navigation;
+  private route = this.props.navigator.route;
```

Comment:
@dgopsq to avoid the TSC error here https://github.com/pagopa/io-app/pull/4056#discussion_r921371758 I have defined a NavigatorProps using a key `(navigator)` and two private (`navigation` and `route`). I don't know if it's best solution what do you think?